symphony-bdk-core/src/test/java/com/symphony/bdk/core/config/BdkConfigLoaderTest.java

```diff
@@ -27,6 +27,9 @@
 import java.util.List;
 import java.util.UUID;
 
+/**
+ * TODO all asserts are inverted: expected value should go first followed by the actual value.
```

Comment:
this is why you use assertj and not unit5 assertions! less confusion :)

Reply:
Indeed, I prefer assertj too

Reply:
can we fix the TODO still (just to avoid leaving some in the code)?